README.md

```diff
@@ -1,43 +1,64 @@
 # Welcome to OpenEBS
 <BR>
-We are an ultra-modern storage target platform, a Hyper-Converged storage software system and a modern Storage Data Fabric natively & tighty integrated into a Kubernetes platform.<BR>
-<BR>
-OpenEBS provides... <BR>
+We are an ultra-modern Block Mode storage platform, a Hyper-Converged software Storage System and a Enterprise Grade virtual NVMe-oF SAN (vSAN) Fabric that is natively & tighty integrated into Kubernetes.<BR>
 <BR>
 
-- Stateful persistent storage volumes for Kubernetes
-- is a 100% Cloud-Native storage solution
-- Delivers a Kubernetes cluster-wide storage fabric that provides containers with access to storage across the entire Kubernetes cluster.
-- Enterprise Grade data management capabilities such as **snapshots, clones and replicated volumes** <BR>
+> [!IMPORTANT]
+> **OpenEBS provides**... <BR>
+> - Stateful persistent Dynamically provisioned storage volumes for Kubernetes
+> - High Performance NVMe-oF storage access optomized for All-Flash Solid State storage media
+> - is a 100% Cloud-Native storage platform
+> - Delivers a Kubernetes cluster-wide vSAN fabric that provides containers/Pods with resillent access to storage across the entire cluster.
+> - Enterprise Grade data management capabilities such as **snapshots, clones, replicated volumes, DIskGroups, Volume Groups, Aggregates, RAID** <BR>
 <BR>
 
-OpenEBS is completely free and open source software. There are also commercial options available for organizations seeking enterprise support and management capabilities. These are provided by third-party vendors. For more information, see https://openebs.io <BR>
+
+### Multiple Storage Engines
+OpenEBS is a Kubernetes provides Persistent Stateful Storage Platform that has **5 core Data Engines**<BR>
+Each Storage Engine provides differnt Capabilities, Flexibility, Resilliance, Data Protections, and Performance features.<BR>
+
+| ID  | Data-Eegines       | Where to create your issues                            |
```

Comment:
Typo: Engines

Reply:
Fixed grammar,

Issue of 1 Single centralized repo was discussed today (4/25/2024) and we agreed on a strategy for this. As noted in issue #3701 and #3709.